### PR TITLE
Add Component Governance manifest

### PR DESCRIPTION
**Pull request by olmisa**
* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.

## Description

Cgmanifest.json added.

## Related PRs or issues

[AB#77893](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/77893)

### CODE REVIEW

File: cgmanifest.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
+{"Registrations":[ 
+     {
+      "Component": {
```
Comment (guperrot) [inline cgmanifest.json:3]: The breakpad submodule is missing.

Comment (olmisa) [reply cgmanifest.json:3]: Fixed

File: cgmanifest.json [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+         "Type": "Maven",
+         "Maven": {
+           "ArtifactId": "gradle",
+           "GroupId": "com.android.tools.build",
+           "Version": "3.3.2"
+         }
+       }
+    },
+    {
+        "Component": {
```
Comment (MatkovIvan) [inline cgmanifest.json:13]: Why do we use not lower case here? Let's harmonize all usages

Comment (olmisa) [reply cgmanifest.json]: Fixed

File: cgmanifest.json [hunk 3]
```diff
@@ -0,0 +1,153 @@
+{"Registrations":[
```
Comment (MatkovIvan) [inline cgmanifest.json:1]: Please format

Comment (olmisa) [reply cgmanifest.json]: Fixed

File: cgmanifest.json [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+        {
+        "component": {
+            "type": "Maven",
+            "maven": {
+            "artifactId": "gradle",
+            "groupId": "com.android.tools.build",
+            "version": "3.3.2"
+            }
+        }
+        },
```
Comment (MatkovIvan) [inline cgmanifest.json:12]: Formatting is still wrong

File: cgmanifest.json [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+                    "groupId": "com.android.support",
+                    "version": "28.0.0"
+                }
+            }
+        },
+        {
+            "component": {
+                "type": "git",
+                "git": {
+                    "repositoryUrl": "https://github.com/google/breakpad.git",
```
Comment (MatkovIvan) [inline cgmanifest.json:147]: We are using MS fork. Please fix it in a separate PR

**Pull request merged**